gulpfile.js

```diff
@@ -0,0 +1,140 @@
+const exec = require('child_process').exec;
+const fs = require('fs');
+const readline = require('readline');
+
+const cwsupload = require('chrome-webstore-upload');
+const eslint = require('gulp-eslint');
+const {google} = require('googleapis');
+const gulp = require('gulp');
+const jeditor = require('gulp-json-editor');
+const zip = require('gulp-zip');
+
+const CWS_ID = 'aojcceglbipehndciapjedoomockgagl';
+
+/**
+ * Checks the code for style errors.
+ */
+gulp.task('lint', () => {
+  return gulp.src(['src/**/*.js', 'gulpfile.js', '!node_modules/**'])
+      .pipe(eslint({
+        baseConfig: require('./.eslintrc.js')
+      }))
+      .pipe(eslint.format())
+      .pipe(eslint.failAfterError());
+});
+
+/**
+ * Updates the Chrome Web Store manifest to point to the latest version of the
+ * scriipt.
+ */
+gulp.task('latest', () => {
+  // Determine the latest version of the script.
+  exec('clasp versions', (err, out) => {
+    if (err) return console.error(err);
+    const line = out.trim().split('\n').pop();
+    const version = line.split(' ').shift();
+
+    // Update the manifest.
+    gulp.src('webstore/manifest.json')
+      .pipe(jeditor((json) => {
+        json.container_info.container_version = version;
+        return json;
+      }))
+      .pipe(gulp.dest('webstore'));
+  });
+});
+
+/**
+ * Bumps the version of the Chrome Web Store manifest.
+ */
+gulp.task('bump', (cb) => {
+  gulp.src('webstore/manifest.json')
+    .pipe(jeditor((json) => {
+      json.version = String(Number(json.version) + 1);
```

Comment:
I don't often see `String` or `Number`, I'd probably write this
`'' + (+json.version + 1)`
But `String(Number(json.version) + 1)` is probably a lot more readable in this case.

https://stackoverflow.com/a/5765401

Reply:
I know we've had this discussion before, but I still prefer that style :-) In this case it starts as a string so we'd also need parseInt thrown in. I think the intent here is just much clearer.

Reply:
OK